docker-run

```diff
@@ -31,11 +31,13 @@ server <- paste(db_conf$hostname, db_conf$path, sep="/")
 # Create connection details using DatabaseConnector utility.
 connectionDetails <- createConnectionDetails(
     dbms=db_conf$scheme, user=db_conf$username, password=db_conf$password,
-    server=server, schema=db_conf$path, port=db_conf$port
+    server=server, port=db_conf$port
```

Comment:
@pavgra Thanks for trying to preserve existing API. Looks fine. it would be ideal for us to add/update docker installation and acceptance tests to CI test suite.